src/spec/helpers.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * this takes an object, converts stringifies it and then parses it back out.
+ * this is useful when fields are Date objects and need to be compared to a
+ * stringified version
+ * @param {*} json
+ * @returns
+ */
+module.exports.parsedJSON = (json) => {
+	return JSON.parse(JSON.stringify(json));
```

Comment:
@eacaps - should we add any null-ish error handling here, so null doesn't get converted to "null" ? Might be overkill for the well-defined use case in the method docs, but it would be nice to get some test coverage for this.